cms/djangoapps/export_course_metadata/management/commands/export_course_metadata_for_all_courses.py

```diff
@@ -26,8 +27,6 @@ def export_course_metadata_for_all_courses():
     """
     Export course metadata for all courses
     """
-    module_store = modulestore()
-    courses = module_store.get_courses()
-
-    for course in courses:
-        export_course_metadata(None, course.id)
+    all_course_keys_qs = CourseOverview.objects.values_list('id', flat=True)
```

Comment:
No filters at all? (I assume this is just same as `expoert_course_metadata`, which is fine).